src/shared/components/mutationTable/MutationTable.tsx

```diff
@@ -111,6 +111,7 @@ export interface IMutationTableProps {
     mrnaExprRankMolecularProfileId?: string;
     discreteCNAMolecularProfileId?: string;
     columns?: ExtendedMutationTableColumnType[];
+    namespaceColumns?: NamespaceColumnConfig;
```

Comment:
why does it make sense for this to be a separate prop, and not just passed in like other columns?

Reply:
The interplay between all MutationMapperStore and MutationMapperTable classes for column definition is rather complicated when looking at the code. I have looked at possibilities to pass the namespace columns as names in the `columns` prop to MutationTable, but this does not work. The fundamental problem is that in order to dynamically create the custom columns the information on type (`string` or `number`) need to be passed along with the column name. This information is used f.i in ResultsViewMutationTable (see comments named `// generate namespace columns`). The new type NamespaceColumnConfig contains this type information.

Reply:
Understood, thank you.